File: app/src/main/java/ru/cool/lwjgl/input/Keyboard.java
package ru.cool.lwjgl.input;

import org.lwjgl.glfw.GLFW;

public class Keyboard {
    public static long currentWindow;

    public static boolean isButtonPress(Buttons pressedKey){
        return GLFW.glfwGetKey(currentWindow, pressedKey.getGlfwKey()) == GLFW.GLFW_PRESS;
    }

    public static boolean isButtonRelease(Buttons releaseKey){
        return GLFW.glfwGetKey(currentWindow, releaseKey.getGlfwKey()) == GLFW.GLFW_RELEASE;
    }

    public static void handleInput(long window){
        currentWindow = window;
    }
}

File: kotlin_app/src/main/kotlin/ru/cool/lwjgl_kotlin/input/Keyboard.kt
package ru.cool.lwjgl_kotlin.input

import org.lwjgl.glfw.GLFW

object Keyboard {
    var currentWindow: Long = 0

    fun isButtonPress(pressedKey: Buttons): Boolean {
        return GLFW.glfwGetKey(currentWindow, pressedKey.keyId) == GLFW.GLFW_PRESS
    }

    fun isButtonRelease(releaseKey: Buttons): Boolean {
        return GLFW.glfwGetKey(currentWindow, releaseKey.keyId) == GLFW.GLFW_RELEASE
    }

    fun handleInput(window: Long) {
        currentWindow = window
    }
}